package.json

```diff
@@ -28,16 +29,17 @@
   "dependencies": {
```

Comment:
[question] Should we update other package version? (for example, as we use in profile/account/authoring/learning MFEs)